src/pycram/yarp_utils/yarp_joint_state_updater.py

```diff
@@ -0,0 +1,324 @@
+import math
+import sys
+from time import sleep
+
+from pycram import World
+from pycram.robot_description import RobotDescription
+from pycram.yarp_utils.yarp_networking import interrupt_and_close
+import yarp
+
+
+class icub_state_updater(yarp.RFModule):
```

Comment:
For the whole class: Add docstrings, typehints for arguments and return 

The python convention for class names is CamelCase, please rename the class

Reply:
done